packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ProjectController.java

```diff
@@ -607,6 +607,39 @@ public ResponseEntity<JsonNode> removeProjectGroupPermissions(
 		}
 	}
 
+	@Operation(summary = "Toggle a project public, or restricted, by ID")
+	@ApiResponses(value = {
+		@ApiResponse(responseCode = "200", description = "Project visibility has been updated", content = {
+			@Content(mediaType = "application/json", schema = @io.swagger.v3.oas.annotations.media.Schema(implementation = UUID.class)) }),
+		@ApiResponse(responseCode = "404", description = "Project could not be found", content = @Content),
+		@ApiResponse(responseCode = "304", description = "The current user does not have delete privileges to this project", content = @Content),
```

Comment:
No 304 (NOT_MODIFIED) is returned from this function.  If one is the message should read, "The current user does not have privileges to modify this project."  Or something like that.